content/en/guides/directory-structure/plugins.md

```diff
@@ -129,13 +129,41 @@ Then we can use it directly in your page components:
 <script>
 export default {
 	async asyncData ({ $axios, params }) {
-	    const  post  = await $axios.$get(`https://api.nuxtjs.dev/posts/${params.id}`)
-	    return { post }
-	  }
+	  const  post  = await $axios.$get(`https://api.nuxtjs.dev/posts/${params.id}`)
+	  return { post }
+	}
 }
 </script>
 ```
 
+Another way to use `axios` without installing the module is by importing `axios` direct in the `<script>` tag.
+
+```js{}[pages/index.vue]
+<script>
+import axios from 'axios'
+
+export default {
+	async asyncData ({ params }) {
+	  const  post  = await axios.$get(`https://api.nuxtjs.dev/posts/${params.id}`)
```

Comment:
Directly using `axios`, it does not support `$` utils

Reply:
oops. - pr: https://github.com/nuxt/nuxtjs.org/pull/884